packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, container, network, host, vip=None):
                 vip=vip,
                 container_type=container,
                 healthcheck_protocol=marathon.Healthcheck.MESOS_HTTP)
+        if network != marathon.Network.HOST and vip is not None:
+            if container == marathon.Container.DOCKER:
+                del self.app['container']['docker']['portMappings'][0]['hostPort']
```

Comment:
deleting port mapping in Bridge mode is a no-op. The system will still generate the Port Mapping. Should we then skip Bridge mode just not to confuse the reader?